llvm/include/llvm/BinaryFormat/ELFRelocs/Sparc.def

```diff
@@ -87,3 +87,12 @@ ELF_RELOC(R_SPARC_GOTDATA_LOX10,  81)
 ELF_RELOC(R_SPARC_GOTDATA_OP_HIX22,  82)
 ELF_RELOC(R_SPARC_GOTDATA_OP_LOX10,  83)
 ELF_RELOC(R_SPARC_GOTDATA_OP,     84)
+ELF_RELOC(R_SPARC_H34,            85)
```

Comment:
Some relocation types are legacy. For example, the two GNU_VT are obsoleted. We should just add ones that are used.